test-plans/ping-versions.template.json

```diff
@@ -0,0 +1,21 @@
+[
+    {
+        "id": "go-libp2p-head",
+        "containerImageID": "FILL THIS IN",
```

Comment:
This is supposed to fill in the hash of the image, right? Do we need to do this? 
It would be nicer if we could just use a tag.

Reply:
Maybe there's a misunderstanding?

The overall idea is this is how you run the current branch against all released versions. This enables interop tests per go-libp2p PR. This extends the defined tests in test-plans by adding this version to the versions already defined.

Once a version is released, the source of truth is the test-plans repo. That defines supported released versions. When we release a version we also push a container image to some container registry (TBD). Then update the test-plans repo to define version A of go-libp2p support X,Y,Z and here's the link to the container image.

This template defines what the current go-libp2p head supports. The `"FILL THIS IN"` is filled in by the Makefile which will put the container image it build for this branch. I should rephrase this to `"This will be filled in automatically"`.